pkg/tinyusb/hw/Makefile

```diff
@@ -1,5 +1,9 @@
 MODULE = tinyusb_hw
 
-SRC = hw_$(CPU).c
+ifneq (,$(filter saml21 samd5x samd21,$(CPU)))
```

Comment:
What about `saml1x`?

Reply:
saml1x doesn't have USB. (and their max clock is limited to 36MHz IIRC).